inference-engine/src/transformations/src/transformations/convert_opset4_to_opset3/convert_sequences_to_sequences_ie.cpp

```diff
@@ -0,0 +1,208 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/convert_opset4_to_opset3/convert_sequences_to_sequences_ie.hpp"
+
+#include <memory>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+#include <ngraph_ops/lstm_sequence_ie.hpp>
+#include <ngraph_ops/gru_sequence_ie.hpp>
+#include <ngraph_ops/rnn_sequence_ie.hpp>
+
+ngraph::pass::ConvertLSTMSequenceMatcher::ConvertLSTMSequenceMatcher() {
+    auto lstm_sequence_ngraph = ngraph::pattern::wrap_type<ngraph::opset4::LSTMSequence>();
+
+    ngraph::matcher_pass_callback callback = [](pattern::Matcher &m) {
+        auto lstm_sequence = std::dynamic_pointer_cast<ngraph::opset4::LSTMSequence>(m.get_match_root());
+        if (!lstm_sequence) {
+            return false;
+        }
+
+        auto W = std::dynamic_pointer_cast<ngraph::opset4::Constant>(
+                lstm_sequence->input_value(4).get_node_shared_ptr());
+        if (!W) {
+            return false;
+        }
+
+        auto R = std::dynamic_pointer_cast<ngraph::opset4::Constant>(
+                lstm_sequence->input_value(5).get_node_shared_ptr());
+        if (!R) {
```

Comment:
please move this checks to pattern:
```suggestion
// JUST EXAMPLE, PLEASE USE NORMAL NAMES FOR VARIABLES
auto input1 = pattern::any_input();
...
auto W = ngraph::pattern::wrap_type<ngraph::opset4::Constant>();
auto lstm_sequence_ngraph = ngraph::pattern::wrap_type<ngraph::opset4::LSTMSequence>(input1, input2, W, R);
```